### PR TITLE
S2GRAPH-5: Add Apache RAT to valid LICENSE errors

### PR DESCRIPTION
**Pull request by SteamShon**
- add apache rat library and create sbt task to run rat on project directory.
- create .rat-excludes to exclude unnecessary files to check license header.

### CODE REVIEW

File: bin/run-rat.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+echo "Base Directory = $base_dir"
+
+if [ -z "$JAVA_HOME" ]; then
+  JAVA="java"
+else
+  JAVA="$JAVA_HOME/bin/java"
+fi
+
+rat_command="$JAVA -jar $base_dir/lib/apache-rat-0.11.jar --dir $base_dir --exclude-file ${rat_excludes_file}"
```
Comment (hsleep) [inline bin/run-rat.sh]: What do you think about adding `-a` option?

Comment (daewon) [inline bin/run-rat.sh]: :+1:

**Comment by SteamShon** [comment]
@hsleep I see your point. I have changed little bit to apply your suggestion.

**Comment by hsleep** [comment]
:+1:

**Pull request closed without merging**